shell/src/main/java/alluxio/cli/bundler/CollectInfo.java

```diff
@@ -61,7 +64,9 @@
 public class CollectInfo extends AbstractShell {
   private static final Logger LOG = LoggerFactory.getLogger(CollectInfo.class);
   private static final String USAGE =
-      "USAGE: collectInfo [--max-threads <threadNum>] [--local] [--help] COMMAND <outputPath>\n\n"
+      "USAGE: collectInfo [--max-threads <threadNum>] [--local] [--help] "
+          + "[--exclude-logs <filename-prefixes>] [--include-logs <filename-prefixes>] "
```

Comment:
The help messages for options are printed at https://github.com/Alluxio/alluxio/blob/c1ea6f3c14812930f014f830d44a79b94574c23e/shell/src/main/java/alluxio/cli/bundler/CollectInfo.java#L154 Basically the description for each option is shown.

For a message that is more verbose, like working examples and how they can be organized to serve use cases, I guess that better suits to be in the document (linked in the help message for CollectInfo). What do you think?

Reply:
ahhh, ok. Now, I see the sub-command help. Then, why does this top level usage help mention start-time and include-logs, etc, when it is only a sub command option?

Reply:
I extracted these option arguments here because they are shared by two options each. Seems it has more confusion that benefit, I will move it back to under the option.